test/integration/model/create.test.js

```diff
@@ -131,7 +131,11 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           }
         }).catch(error => {
           expect(error).to.be.instanceof(Sequelize.UniqueConstraintError);
-          expect(error.fields).to.include({ username: 'gottlieb' });
+          if (dialect === 'mysql') {
```

Comment:
Its failing for `sqlite`, a better solution would be to have fields return same object in `sqlite` as other dialects

Reply:
I think that is out of the scope of the PR. The changes I made, only process the contents of the where clause of the findOrCreate function, which is supposed to be independent of the DB engine. I also saw conditional assertions based on the dialect in other tests in the same file.

Reply:
May be SQLite doesnt support returning values in error, then I have never worked with SQLite in production. Please change the check to `dialect === sqlite` and assert for plain array with `username` in that case, keep current assertion for other dialects